internal/notif/pushover/client.go

```diff
@@ -60,16 +72,77 @@ func (c *Client) Send(entry model.NotifEntry) error {
 		return err
 	}
 
-	_, err = pushover.New(token).SendMessage(&pushover.Message{
-		Title:     string(title),
-		Message:   string(body),
-		Priority:  c.cfg.Priority,
-		Sound:     c.cfg.Sound,
-		URL:       c.meta.URL,
-		URLTitle:  c.meta.Name,
-		Timestamp: time.Now().Unix(),
-		HTML:      true,
-	}, pushover.NewRecipient(recipient))
-
-	return err
+	cancelCtx, cancel := context.WithCancelCause(context.Background())
+	timeoutCtx, _ := context.WithTimeoutCause(cancelCtx, *c.cfg.Timeout, errors.WithStack(context.DeadlineExceeded)) //nolint:govet // no need to manually cancel this context as we already rely on parent
+	defer func() { cancel(errors.WithStack(context.Canceled)) }()
+
+	form := url.Values{}
+	form.Add("token", token)
+	form.Add("user", recipient)
+	form.Add("title", string(title))
+	form.Add("message", string(body))
+	form.Add("priority", strconv.Itoa(c.cfg.Priority))
+	if c.cfg.Sound != "" {
+		form.Add("sound", c.cfg.Sound)
+	}
+	if c.meta.URL != "" {
+		form.Add("url", c.meta.URL)
+	}
+	if c.meta.Name != "" {
+		form.Add("url_title", c.meta.Name)
+	}
+	form.Add("timestamp", strconv.FormatInt(time.Now().Unix(), 10))
+	form.Add("html", "1")
```

Comment:
Might make sense to log the request body under Debug. (Maybe with token sanitized?)